src/webapi/events/pointer.rs

```diff
@@ -0,0 +1,483 @@
+    ///
+    /// [(JavaScript docs)](https://developer.mozilla.org/en-US/docs/Web/API/PointerEvent/tiltX)
+    // https://w3c.github.io/pointerevents/#dom-pointerevent-tiltx
+    #[inline]
+    fn tilt_x( &self ) -> i32 {
+        js!(
+            return @{self.as_ref()}.tiltX;
+        ).try_into().unwrap()
+    }
+
+    /// Returns the angle, in the range of [-90, 90] degrees, between the X-Z plane and the plane
+    /// containing the transducer (e.g. pen stylus) and the X axis.
+    ///
+    /// [(JavaScript docs)](https://developer.mozilla.org/en-US/docs/Web/API/PointerEvent/tiltY)
+    // https://w3c.github.io/pointerevents/#dom-pointerevent-tilty
+    #[inline]
+    fn tilt_y( &self ) -> i32 {
+        js!(
+            return @{self.as_ref()}.tiltY;
+        ).try_into().unwrap()
+    }
+
+    /// Returns the clockwise rotation, in the range of [0, 359] degrees, of
+    /// the transducer (e.g. pen stylus) around it's own major axis
+    ///
+    /// [(JavaScript docs)](https://developer.mozilla.org/en-US/docs/Web/API/PointerEvent/twist)
+    // https://w3c.github.io/pointerevents/#dom-pointerevent-twist
+    #[inline]
+    fn twist( &self ) -> i32 {
+        js!(
+            return @{self.as_ref()}.twist;
+        ).try_into().unwrap()
+    }
+
+    /// Indicates the device type that caused the event.
+    ///
+    /// [(JavaScript docs)](https://developer.mozilla.org/en-US/docs/Web/API/PointerEvent/pointerType)
+    // https://w3c.github.io/pointerevents/#dom-pointerevent-pointertype
+    #[inline]
+    fn pointer_type( &self ) -> String {
+        js!(
+            return @{self.as_ref()}.pointerType;
+        ).try_into().unwrap()
+    }
+
+    /// Indicates if the pointer represents the primary pointer of this pointer type
+    ///
+    /// [(JavaScript docs)](https://developer.mozilla.org/en-US/docs/Web/API/PointerEvent/isPrimary)
+    // https://w3c.github.io/pointerevents/#dom-pointerevent-isprimary
+    #[inline]
+    fn is_primary( &self ) -> bool {
+        js!(
+            return @{self.as_ref()}.isPrimary;
+        ).try_into().unwrap()
+    }
+
+    /// Returns the sequence of all `PointerEvent` instances that were coalesced into the dispatched `PointerMoveEvent`.
+    ///
+    /// [(JavaScript docs)](https://developer.mozilla.org/en-US/docs/Web/API/PointerEvent/getCoalescedEvents)
+    // https://w3c.github.io/pointerevents/extension.html#dom-pointerevent-getcoalescedevents
+    #[inline]
+    fn get_coalesced_events( &self ) -> Vec<PointerEvent> {
+        js!(
+            return @{self.as_ref()}.getCoalescedEvents();
+        ).try_into().unwrap()
+    }
+
+    /// Indicates the mouse button that fired this event. A None value indicates no change since the last PointerEvent.
+    ///
+    /// [(JavaScript docs)](https://developer.mozilla.org/en-US/docs/Web/API/MouseEvent/button)
+    // https://w3c.github.io/pointerevents/#the-button-property
+    fn button_pointer( &self ) -> Option<MouseButton> {
```

Comment:
Hmm... I think that *probably* it would be better to merge this with the `pointer` method, but as you've said that would be a breaking change. I'm fine with merging this as-is for now, however can you attach `#[cfg(feature = "experimental_features_which_may_break_on_minor_version_bumps")]` to this function? (That's how we mark totally ustable functionality; if we're going to remove this then there is no point in pretending that it's stable.)